docs/building.md

```diff
@@ -96,12 +96,26 @@ interchangeable right now.
 * Experimental CMake generation is available to facilitate use of other IDEs such as [CLion](https://www.jetbrains.com/clion/). If `clion` is available inside `$PATH`, `xb devenv` will start it. Otherwise `build/CMakeLists.txt` needs to be generated by invoking `xb premake --devenv=cmake` manually.
 
 Clang-9 or newer should be available from system repositories on all up to date distributions.
-You will also need some development libraries. To get them on an Ubuntu system:
+You will also need some development libraries. 
 
+##### Ubuntu
 ```bash
 sudo apt-get install libgtk-3-dev libpthread-stubs0-dev liblz4-dev libx11-dev libx11-xcb-dev libvulkan-dev libsdl2-dev libiberty-dev libunwind-dev libc++-dev libc++abi-dev
 ```
 
+#### Arch
+```bash
+sudo pacman -Syu
+sudo pacman -S glew gtk3 libx11 lz4 clang git python vulkan-headers
+```
+libpthread-stubs needs to be compiled from the AUR:
+https://aur.archlinux.org/packages/libpthread-stubs/
+#### Fedora (untested)
```

Comment:
For more consistent Markdown parsing by different viewer apps, I think it's better to separate all headings, as well as all code blocks, with blank lines on both sides.